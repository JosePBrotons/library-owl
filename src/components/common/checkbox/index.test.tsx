import React from 'react';
import renderer from 'react-test-renderer';
import Checkbox from '.';

test('renders input component correctly', () => {
    const props = {
        checked: false,
        description: 'I agree with the terms and conditions',
        onPress: jest.fn,
    };
    const inputCmp = renderer.create(<Checkbox {...props} />).toJSON();
    expect(inputCmp).toMatchSnapshot();
});
