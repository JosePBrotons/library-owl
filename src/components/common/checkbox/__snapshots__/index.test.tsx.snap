// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders input component correctly 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "flexDirection": "row",
      "justifyContent": "center",
      "marginVertical": 15,
      "opacity": 1,
      "width": "80%",
    }
  }
>
  <Text
    allowFontScaling={false}
    style={
      Array [
        Object {
          "color": undefined,
          "fontSize": 12,
        },
        Object {
          "color": "rgba(57, 172, 233, 1)",
          "fontFamily": "Nunito-Regular",
          "fontSize": 22,
          "lineHeight": 30,
        },
        Object {
          "fontFamily": "FontAwesome",
          "fontStyle": "normal",
          "fontWeight": "normal",
        },
        Object {},
      ]
    }
  >
    ï‚–
  </Text>
  <Text
    ellipsizeMode="tail"
    numberOfLines={1}
    style={
      Object {
        "color": "rgba(106, 106, 106, 1)",
        "fontFamily": "Nunito-Regular",
        "fontSize": 14,
        "lineHeight": 22,
        "marginLeft": 10,
        "textAlign": "center",
      }
    }
  >
    I agree with the terms and conditions
  </Text>
</View>
`;
